OrbitGrpcProtos/capture.proto

```diff
@@ -61,6 +61,10 @@ message FunctionCall {
   repeated uint64 registers = 8;
 }
 
+message IntrospectionCall {
+  FunctionCall function_call = 1;
```

Comment:
In general this is not a function call, it should be called `IntrospectionScope` and not reuse `FunctionCall`.

Reply:
Done.